support/modules/single-file

```diff
@@ -21,39 +21,40 @@ case "$STATE" in
         ;;
 
     ArtifactInstall)
-        dest_dir=$(cat $dest_dir_file)
-        filename=$(cat $filename_file)
+        dest_dir=$(cat "${dest_dir_file}")
+        filename=$(cat "${filename_file}")
```

Comment:
It is more important to put quotes around `$(cat ...)` than around `filename_file`, since we already know those don't have spaces. The content of them may have though.

This goes for all the occurrences of `$(...)`.

Reply:
Looks like you're right, it's not needed in the specific case where the `$()` is being assigned to a variable.